packages/manager/modules/carbon-calculator/src/dashboard/routing.js

```diff
@@ -0,0 +1,14 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dashboard', {
+    url: '/dashboard/',
+    component: 'carbonCalculatorDashboard',
+    resolve: {
+      currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
+        $state.href($state.current.name, $transition$.params()),
+      dashboardLink: /* @ngInject */ ($state) => $state.href('app.dashboard'),
+      serviceName: /* @ngInject */ ($transition$) =>
```

Comment:
I'm not sure that is needded, because carbonCalculator is not the concern of a particular service

Reply:
I will fix/clean in my next PR if you are OK. Here, the goal is just to generate a new micro-app with the scaffolding tools